packages/electron-publish/src/BintrayPublisher.ts

```diff
@@ -54,26 +54,40 @@ export class BintrayPublisher extends HttpPublisher {
     }
   }
 
-  protected async doUpload(fileName: string, dataLength: number, requestProcessor: (request: ClientRequest, reject: (error: Error) => void) => void) {
+  protected async doUpload(fileName: string, arch: string, dataLength: number, requestProcessor: (request: ClientRequest, reject: (error: Error) => void) => void) {
     const version = await this._versionPromise
     if (version == null) {
       debug(`Version ${this.version} doesn't exist and is not created, artifact ${fileName} is not published`)
       return
     }
 
     let attemptNumber = 0
+
+    const options: RequestOptions = {
+      hostname: "api.bintray.com",
+      path: `/content/${this.client.owner}/${this.client.repo}/${this.client.packageName}/${version.name}/${fileName}`,
+      method: "PUT",
+      headers: {
+        "Content-Length": dataLength,
+        "X-Bintray-Override": "1",
+        "X-Bintray-Publish": "1",
+        "X-Bintray-Debian-Architecture": arch
+      }
+    }
+
+    if (this.client.distribution) {
+      options.headers = options.headers || {};
```

Comment:
I added these lines because TypeScript would complain that `options` might be `undefined`. @develar Is there a better way to address that?